tutorials/energy/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Energy consumption data tutorial
+excerpt: Learn how to query energy consumption data
+products: [cloud]
+keywords: [tutorials, energy, learn]
+tags: [tutorials, beginner]
+layout_components: [next_prev_large]
+content_group: Analyze energy consumption data
+---
+
+# Analyze energy consumption data
+
+When you are planning to switch to a photovoltaic system it isn’t easy, even
```

Comment:
I wonder if we work in the more vernacular "rooftop solar" phrasing here, just in case people don't immediately recognise what we mean by a PV system?

Reply:
fixed